.eslintrc.js

```diff
@@ -1,5 +1,10 @@
 module.exports = {
 	extends: [
 		'@nextcloud',
-	]
+	],
+	rules: {
```

Comment:
TODO: Fix these soon

Reply:
in another PR (where we also fix the indentation and other problems these rules avoided)